components/chassis.py

```diff
@@ -26,14 +27,13 @@ class SwerveModule:
     STEER_GEAR_RATIO = (14 / 50) * (10 / 60)
     WHEEL_CIRCUMFERENCE = 4 * 2.54 / 100 * math.pi
 
-    METRES_TO_DRIVE_UNITS = 2048 / WHEEL_CIRCUMFERENCE / DRIVE_GEAR_RATIO
-    DRIVE_SENSOR_TO_METRES = 1 / METRES_TO_DRIVE_UNITS
+    DRIVE_MOTOR_REV_TO_METRES = WHEEL_CIRCUMFERENCE * DRIVE_GEAR_RATIO
+    METRES_TO_DRIVE_COUNTS = FALCON_CPR / DRIVE_MOTOR_REV_TO_METRES
+    DRIVE_COUNTS_TO_METRES = DRIVE_MOTOR_REV_TO_METRES / FALCON_CPR
 
-    STEER_SENSOR_TO_RAD = math.tau / 2048 * STEER_GEAR_RATIO
-    STEER_RAD_TO_SENSOR = 1 / STEER_SENSOR_TO_RAD
-
-    SLEW_CRUISE_VELOCITY = 400  # rpm
-    CRUISE_ACCELERATION = 200  # rpm/s
```

Comment:
These two constants were unused. I believe these were a copy-paste holdover from when we were testing the Smart Motion stuff on our previous swerve iteration.